ddtrace/contrib/aiomysql/patch.py

```diff
@@ -83,6 +88,10 @@ async def _trace_method(self, method, resource, extra_tags, *args, **kwargs):
             # set analytics sample rate
             s.set_tag(ANALYTICS_SAMPLE_RATE_KEY, config.aiomysql.get_analytics_sample_rate())
 
+            dbm_propagator = getattr(config.aiomysql, "_dbm_propagator", None)
```

Comment:
Using `core.dispatch()` from `ddtrace.internal.core` here would keep DBM's product logic separate from the instrumentation logic that this file focuses on.